src/Maestro/SubscriptionActorService/MaestroBarClient.cs

```diff
@@ -173,10 +168,44 @@ public async Task<IEnumerable<Build>> GetBuildsAsync(string repoUri, string comm
             return builds.Select(b => ToClientModelBuild(b));
         }
 
+        public async Task<IEnumerable<Asset>> GetAssetsAsync(string name = null,
+            string version = null, int? buildId = null, bool? nonShipping = null)
```

Comment:
Nit: move each parameter to its own line

Reply:
Yep, I'll fix that, forgot to fix this when I moved this method from the "Unneeded APIs region" to the "I really need this to work" region.